rollup/internal/config/relayer.go

```diff
@@ -70,8 +70,14 @@ type RelayerConfig struct {
 type GasOracleConfig struct {
 	// MinGasPrice store the minimum gas price to set.
 	MinGasPrice uint64 `json:"min_gas_price"`
-	// GasPriceDiff store the percentage of gas price difference.
+	// GasPriceDiff is the minimum gas price difference to update gas oracle.
```

Comment:
This is still the percentage of the gas price difference, right?

Reply:
Yes. Updated the comment again in [da6b00e](https://github.com/scroll-tech/scroll/pull/1305/commits/da6b00ed52086f290dce411530c36133c1260b23).